src/TrigScint/SimQIE.cxx

```diff
@@ -0,0 +1,164 @@
+/**
+ * @file SimQIE.cxx
+ * @author Niramay Gogate, Texas Tech University
+ */
+
+#include<iostream>
+#include <exception>
+#include"TMath.h"
+#include"TrigScint/SimQIE.h"
+#include "Framework/Exception/Exception.h"
+
+namespace trigscint {
+
+  SimQIE::SimQIE() {}
+
+  SimQIE::SimQIE(float PD, float SG, uint64_t seed=0) {
```

Comment:
Why is this not using the RandomSeedService?

Reply:
The `SimQIE` object is initialized in the `TrigScintQIEDigiProducer` with the seed provided by RandomSeedService. The idea is to initialize the random number only at the beginning of the simulation. Defining the constructor this way allows us to control the random number right from the producer.

Reply:
does the randomSeedService support a deterministic seed?  If so, I don't see why this is needed.

Reply:
okay, Niramay has convinced me that this is done correctly.  There is just pass through of the random seed, which is received from the RandomNumberSeedService.  So, I think this should be left as is.

Reply:
Yes this is fine.